lib/puppet/type/cron.rb

```diff
@@ -18,29 +18,29 @@
 
     Example:
 
-        cron { logrotate:
-          command => "/usr/sbin/logrotate",
-          user    => root,
-          hour    => 2,
-          minute  => 0
+        cron { 'logrotate':
+          command => '/usr/sbin/logrotate',
+          user    => 'root',
+          hour    => '2',
+          minute  => '0',
```

Comment:
Well, if it's just for consistency with strings like `'*/10'`, I'm fine with that, but in that case the array of numbers below should be an array of quoted numbers too.